lib/pid_ng/include/PID.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 202 COGIP Robotics association <cogip35@gmail.com>
+// This file is subject to the terms and conditions of the GNU Lesser
+// General Public License v2.1. See the file LICENSE in the top level
+// directory for more details.
+
+/// @ingroup     lib_pid
+/// @{
+/// @file
+/// @brief       PID implementation
+/// @author      Gilles DOFFE <g.doffe@gmail.com>
+
+#pragma once
+
+#include <etl/limits.h>
+
+namespace cogip {
+
+namespace pid {
+
+/// PID
+class PID {
+    public:
+        /// Constructor.
+        PID(
+            double kp = 0.0,                                                ///< [in] proportional gain
+            double ki = 0.0,                                                ///< [in] integral gain
+            double kd = 0.0,                                                ///< [in] derivative gain
+            double integral_term_limit = etl::numeric_limits<double>::max() ///< [in] integral term limit
+        ) : kp_(kp), ki_(ki), kd_(kd), integral_term_(0), integral_term_limit_(integral_term_limit), previous_error_(0) {};
+
+        /// Return proportional gain.
+        double kp(void) const { return kp_; };
```

Comment:
`void` is not required when you don't have parameters, it is useless.

Reply:
done